.github/workflows/container-validation-dynamo.yml

```
@@ -91,3 +91,11 @@ jobs:
         with:
           name: Event File
           path: ${{ github.event_path }}
+
+  # Upload metrics for this workflow and all its jobs
+  upload-workflow-metrics:
+    needs: [build-test]  # Wait for main job to complete
+    if: always()  # Always run, even if other jobs fail
+    continue-on-error: true  # Don't fail the workflow if metrics upload fails
```

actionlint (1.7.8)

when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "upload-workflow-metrics"

(syntax-check)